sky/clouds/service_catalog/data_fetchers/fetch_azure.py

```diff
@@ -26,6 +26,13 @@
     # 'WestUS3',   # WestUS3 pricing table is broken as of 2021/11.
 ]
 
+# Exclude the following regions as they do not have ProductName in the
+# pricing table.
```

Comment:
nit: Could you add the link to the related issue, just to make the context clearer?

Reply:
Done. Thanks!